packages/ember-glimmer-template-compiler/lib/plugins/transform-attrs-into-args.js

```diff
@@ -38,7 +38,10 @@ TransformAttrsToProps.prototype.transform = function TransformAttrsToProps_trans
   traverse(ast, {
     PathExpression(node) {
       if (node.parts[0] === 'attrs') {
-        return b.path(node.original.substr(6));
+        let path = b.path(node.original.substr(6));
+        path.original = `@${path.original}`;
```

Comment:
I am concerned about the implications here, that users can start using the `@foo` syntax. We definitely should be preventing this until we have an RFC proposing it.